### ISSUE TITLE
Make CNN Model

### ISSUE THREAD

**Issue by LoganWatson1400** [LoganWatson1400/Brady_Hackathon#4]
in the make model file, we need to make a base CNN model.
if you want an example, this is a project to classify emotion pictures i made before
[https://github.com/LoganWatson1400/Emotion_Classificaton](url)

Things to concider
you can use a pre trained model to form a base for image classification
what you do, is load a pre trained model from [https://keras.io/api/applications/](url) or another source you can find
then, you lock all but the last couple layers in the model, and then retrain those layers on our data

Pre-trained CNN models save time, reduce computational costs, and improve accuracy by leveraging learned features from large datasets. They enable efficient transfer learning, requiring less data and training for high performance in tasks like image classification and object detection.

this is a segment of my emotion CNN using this
# Fine-tuning a pre-trained model (Xception)
  def build_pretrained_xception(train_set):
      base_model = tf.keras.applications.Xception(weights='imagenet', include_top=False, input_shape=(*img_size, 3))
      base_model.trainable = False
  
      model = tf.keras.Sequential([
          base_model,
          tf.keras.layers.GlobalAveragePooling2D(),
          tf.keras.layers.Dropout(0.5),
          tf.keras.layers.Dense(len(train_set.class_names), activation='softmax')
      ])
      model.compile(optimizer='adam',
                    loss='categorical_crossentropy',
                    metrics=['accuracy'])
      return model
